### DISCUSSION TITLE
Coding Style for Memtest86+

### DISCUSSION THREAD

**Discussion by 01e3** [memtest86plus/memtest86plus#163] [Enhancement]
Having a comprehensive Coding Style becomes a must-have for a project with multiple developers and being open for external contributions.

The discussion in https://github.com/memtest86plus/memtest86plus/commit/f265d1f1c50aa57116d5ac667be8003f706f9e0d suggests one of the areas we may want to cover first.

Easy to request, not that easy to implement, especially when comes to deciding how to decide, but I'm filling this bug to track.

**Comment by martinwhitaker** [comment]
We already have one, albeit very brief. See the doc/README_DEVEL.md file.

Note that the automatically generated internal documentation is work in progress. system/vmem.h shows the style I'm aiming for.

**Comment by 01e3** [comment]
Oh, indeed. I updated the bug a little bit then.
